README.md

```diff
@@ -14,7 +14,7 @@ To contibute:
 2. Install nodejs (we will be using npm)
 3. Open a terminal and `cd` into backend repo and run `npm i` and then `npm run dev`
 4. Open another terminal and `cd` into the frontend repo. Run `npm i` and then `npm run dev`
-5. Open your browser and go to `http://127.0.0.1:5173/` to open the app (temporary `http://127.0.0.1:5173/abc` to see timer)
+5. Open your browser and go to `http://127.0.0.1:5100/` to open the app (temporary `http://127.0.0.1:5100/abc` to see timer)
```

Comment:
@eyejabb Thanks for catching this, can you also please update it on all the other repos as well please?